pages/login.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+  <div
+    id="login"
+    class="flex flex-grow container mx-auto p-3 justify-center items-start"
+  >
+    <log-in-card @signUpClick="$router.push('/signup')" />
```

Comment:
Do we want to use the `<log-in-card ... />` style here for some particular reason? I don't know if this was something Nuxt required for `pages` or not, but I feel like the way that we used the `<LogInCard ... />` style, like in other components, makes it more clear that it's a custom element.

Reply:
Totally missed that. I'll switch it over!